src/protocol-democracy.md

```diff
@@ -98,3 +98,60 @@ If we want to relax this in the future, we would need to come up with a way to h
 
 In order to determine if a vote is passing, we use Positive Turnout Bias. In addition we enforce a minimum turnout of 5%.
 
+### Documentation
```

Comment:
As this is a tutorial it doesn't belong on this page. It should be its own page in the tutorial section and just linked from here